# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/large-scaled-mask.html
# Commit: 95d3fbb0f317
# Full Hash: 95d3fbb0f317131fe566fdb1838f5cf49266a63f
# Author: Jeff Muizelaar <jmuizelaar@mozilla.com>
# Date: 2021-06-04 15:42:19
# Description:
#   Bug 1712195. Use SafeIntersect. r=aosmond
#   
#   This uses the new SafeIntersect function
#   to avoid empty rects that overflow from leaking
#   through.
# ==============================================================================

diff -r 6a301646ecc9 -r 95d3fbb0f317 gfx/tests/crashtests/large-scaled-mask.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/large-scaled-mask.html	Fri Jun 04 12:35:42 2021 +0000
@@ -0,0 +1,8 @@
+<style>
+* {
+  scale: 48% 41467;
+  border: solid !important;
+  filter: url(#x);
+  -webkit-mask-image: url(data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=)
+}
+</style>
